easybuild/easyconfigs/i/ifort/ifort-2016.0.109-GCC-4.9.3-2.25.eb

```diff
@@ -0,0 +1,31 @@
+# This is an easyconfig file for EasyBuild, see http://hpcugent.github.io/easybuild
+
+name = 'ifort'
+version = '2016.0.109'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['parallel_studio_xe_%(version_major)s_composer_edition_for_fortran.tgz']
+
+checksums = ['bce7f6a71f7e44f67956197501d00b7c']
+
+gcc = 'GCC'
```

Comment:
as pointed out by @ocaisa in #2214, this should be `GCCcore`, and `binutils` should be included as well as a dep